2-library/6core-logging/pom.xml

```
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.6core.lib</groupId>
+    <artifactId>6core-bom</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <groupId>com.6core.lib</groupId>
+  <artifactId>lib-logging</artifactId>
```

you have `lib` postfix in groupdId. Why do we need to have it in artifactId as well?  no need for duplication here.

```suggestion
  <groupId>com.6core.lib</groupId>
  <artifactId>lib-logging</artifactId>
```